packages/terra-framework-docs/src/terra-dev-site/doc/data-grid/FlowsheetDataGrid.4/Examples.6/ColumnHeadersHidden.jsx

```diff
@@ -0,0 +1,134 @@
+import React, { useState, useCallback } from 'react';
+import { FlowsheetDataGrid } from 'terra-data-grid';
+
+const DefaultFlowsheetDataGrid = () => {
```

Comment:
Shouldn't this be ColumnHeadersHidden instead of DefaultFlowsheetDataGrid?

Reply:
Fixed with 4e96fbb